Firmware/Marlin_main.cpp

```diff
@@ -8321,9 +8321,9 @@ SERIAL_PROTOCOLPGM("\n\n");
     */
 	case 861:
 		if (code_seen('?')) { // ? - Print out current EEPROM offset values
-			uint8_t cal_status = calibration_status_pinda();
 			int16_t usteps = 0;
-			cal_status ? SERIAL_PROTOCOLLN("PINDA cal status: 1") : SERIAL_PROTOCOLLN("PINDA cal status: 0");
+			SERIAL_PROTOCOL("PINDA cal status: ");
+			SERIAL_PROTOCOLLN(calibration_status_pinda());
```

Comment:
ummm... can't this be the PGM variant?

Reply:
I think so. We can save a lot of SRAM (~50 bytes) but the flash usage will increase about 60 bytes if I remember correctly.

Reply:
Flash increase... That sounds wrong. What if you use a printf_P?

Reply:
`printf_P(PSTR("PINDA cal status: %u\n"), calibration_status_pinda() ? 1 : 0);` sounds correct to me. I hope it's optimal. In any case, I'll always trade flash for ram. Issues caused by ram can be very deceiving, while a lack of flash means the build will fail (obviously trivial to debug).

Reply:
Please don't commit to this. I have an awesome improvement. I'll squash this commit into the original Pinda status commit

Reply:
See https://github.com/prusa3d/Prusa-Firmware/pull/3378/commits/aa7aefd6f844c470f1616d9a88632dd0827417ef